src/main/java/com/google/growpod/data/Garden.java

```diff
@@ -0,0 +1,94 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.growpod.data;
+
+/** Garden data class. */
+public class Garden {
+
+  /** A unique id. */
+  private String id;
+
+  private String name;
```

Comment:
nit: This is missing a comment. It's pretty self explanatory so it's not a big deal that it's not annotated, however it is somewhat inconsistent with the rest of the file which is all annotated.

Reply:
Will add notes in all similar places to make everything consistent.